androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Checkbuttons.kt

```diff
@@ -97,44 +99,110 @@ fun <E : Enum<*>> EnumCheckBoxLazyGrid(
-        content = {
-            items(choices.size) { index ->
-                var isChecked by rememberSaveable { mutableStateOf(choices[index] in currentSelection) }
-                Row(
-                    verticalAlignment = Alignment.CenterVertically,
-                    modifier = Modifier.padding(4.dp)
-                ) {
-                    Box(
-                        modifier = Modifier.background(circleBgColor, CircleShape)
+            }
+        )
+    } else {
+        LazyVerticalGrid(
+            columns = gridLayout,
+            content = {
+                items(choices.size) { index ->
+                    var isChecked by rememberSaveable { mutableStateOf(choices[index] in currentSelection) }
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.padding(4.dp)
                     ) {
-                        Checkbox(
-                            checked = isChecked,
-                            onCheckedChange = { checked ->
-                                isChecked = checked
-                                when(checked) {
-                                    false -> currentSelection.remove(choices[index])
-                                    true -> currentSelection.add(choices[index])
+                        Box(
+                            modifier = Modifier.background(colorCheckBoxBeta, CircleShape)
+                        ) {
+                            Checkbox(
+                                checked = isChecked,
+                                onCheckedChange = { checked ->
+                                    isChecked = checked
+                                    when (checked) {
+                                        false -> currentSelection.remove(choices[index])
+                                        true -> currentSelection.add(choices[index])
+                                    }
+                                    onSelectionChanged(currentSelection.toSet())
                                 }
-                                onSelectionChanged(currentSelection.toSet())
-                            }
+                            )
+                        }
+                        Text(
+                            text = labels[choices[index]] ?: choices[index].name,
+                            textAlign = TextAlign.Start,
+                            modifier = Modifier.padding(start = 8.dp),
+                            fontSize = textFontSize.sp
                         )
                     }
-                    Text(
-                        text = labels[choices[index]] ?: choices[index].name,
-                        textAlign = TextAlign.Start,
-                        modifier = Modifier.padding(start = 8.dp),
-                        fontSize = textFontSize.sp
-                    )
                 }
             }
+        )
+    }
+}
+
+
+@Composable
+fun LabeledCheckbox(
```

Comment:
Nice to have a composable for a single checkbox! However, why is is not used in the other checkox-set-composables in this file?